copulas/univariate/base.py

```diff
@@ -184,6 +184,9 @@ def to_dict(self):
         if not self.fitted:
             return result
 
+        if isinstance(self, Univariate):
```

Comment:
Temporary workaround to make saving/loading instances of the Univariate base class work. We need to fix it properly at some point, but for now, I want to focus on writing the tutorials.